client-s3/src/test/resources/file-structure/in-the-kitchen-with-mikepaw/dinner/main-butter-chicken

```diff
@@ -0,0 +1,27 @@
+For Marinating the Chicken:
```

Comment:
:chicken:

Reply:
Its a good butter chicken recipe.

Also I discovered relatively recently that butter chicken is basically tomato bisque...